sql/moz-fx-data-shared-prod/search_terms/sanitization_job_languages/view.sql

```diff
@@ -0,0 +1,11 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.search_terms.sanitization_job_languages`
+AS
+SELECT
+  started_at as job_start_time,
+  lang.key AS language_code,
+  CAST(lang.value AS int) AS search_term_count
+FROM
+  `moz-fx-data-shared-prod.search_terms_derived.sanitization_job_metadata`
+CROSS JOIN
+  UNNEST(mozfun.json.js_extract_string_map(approximate_language_proportions_json)) AS lang
```

Comment:
Per https://mozilla.github.io/bigquery-etl/mozfun/json/#js_extract_string_map-udf this is a JS UDF that will not have great performance, but I don't expect that to be a significant issue for the scale of data we have here. If were to become a problem, then we can consider materializing this logic to a derived table and updating the view to point there instead.

Reply:
^^ Definitely. Because we're imagining this running daily on a dictionary with maybe two dozen keys in it, I figured it was okay for now. But we could totally do that.

Another option is to insert these sums into this new table from the job that creates the JSON in the first place.